app/views/mortgage_calculator/stamp_duties/_form_step2.html.erb

```diff
@@ -70,7 +70,7 @@
 <div class="stamp-duty__info-column">
   <div class="callout callout--tip">
     <span class="callout__icon" aria-hidden="true">?</span>
-    <div class="stamp-duty__info-subheading callout__heading"><%= I18n.t("stamp_duty.next_steps.learn_more.title") %></div>
+    <h3 class="stamp-duty__info-subheading callout__heading"><%= I18n.t("stamp_duty.next_steps.learn_more.title") %></h3>
```

Comment:
Similar point to above re the change to h3 though in this instance this probably shouldn't be a div! (Again I left this as is in the previous work so as not to alter the existing markup unduly).